cfme/tests/cloud/test_provisioning.py

```diff
@@ -66,8 +71,11 @@ def testing_instance(request, setup_provider, provider, provisioning, vm_name):
 
 
 @pytest.fixture(scope="function")
-def vm_name(request):
-    vm_name = 'test_image_prov_{}'.format(fauxfactory.gen_alphanumeric())
+def vm_name(request, provider):
+    if provider.key == 'gce_central':
```

Comment:
We cannot decide this based on `provider.key`, it has to be done the same way as above, using `isinstance`.